### PR TITLE
Print where kernel is installed

### PR DESCRIPTION
**Pull request by teto**
not sure if you prefer a debug statement or a print.
I find it useful to know where the files are installed

### CODE REVIEW

**Comment by vaibhavsagar** [comment]
Debug statement please, happy to merge after you make that change.

**Comment by teto** [comment]
cool. How can I log a debug statement ? I've just found a getLogger but I can't find any log level or example. Where should I look ?

**Comment by vaibhavsagar** [comment]
What I had in mind is that you can reuse the `kernelSpecDebug` flag to determine whether or not you output the kernel install directory.

**Comment by teto** [comment]
kernelSpecDebug serves a different purpose ? would it be worth introducing a proper logger with different loglevels like Katip  https://github.com/Soostone/katip ?

**Comment by vaibhavsagar** [comment]
No, I think that's overkill. If you want to unconditionally print out the kernelspec installation path then that's fine, but it seems somewhat inconsistent to do that for the jupyterlab kernelspec but not the classic kernelspec.

**Comment by teto** [comment]
Got it, I dont like though that the kernel flag is used with a different meaning to tell ihaskell what to do. What about a verbosity flag `-v` ?
I've improved the flake (needed to bring jupyter for the tests too). 
I've added entries that matches the release.nix built environments and I need your approval on the name, I hesitate between ihaskellEnv / ihaskell-env / ihaskell-jupyter or maybe jupyter-with-ihaskell etc. currently it's:
```
        ihaskell-env = mkEnv ghcDefault;
        ihaskell-env-8107 = mkEnv ghc8107;
```

**Comment by teto** [comment]
Got it, I dont like though that the kernel flag is used with a different meaning to tell ihaskell what to do. What about a verbosity flag `-v` ?
I've improved the flake (needed to bring jupyter for the tests too). 
I've added entries that matches the release.nix built environments and I need your approval on the name, I hesitate between ihaskellEnv / ihaskell-env / ihaskell-jupyter or maybe jupyter-with-ihaskell etc. currently it's:
```
        ihaskell-env = mkEnv ghcDefault;
        ihaskell-env-8107 = mkEnv ghc8107;
```

**Comment by vaibhavsagar** [comment]
Can you make a different PR for these changes?

File: src/IHaskell/IPython.hs
Comment (vaibhavsagar) [inline src/IHaskell/IPython.hs:191]: Can you use `putStrLn` here instead of `print`? Also `(++)` instead of `(<>)` for GHC 8.0.

**Comment by teto** [comment]
> Can you make a different PR for these changes?

sorry that was my initial intent and then I got lazy. I opened  a PR with just the nix changes at https://github.com/gibiansky/IHaskell/pull/1306 . Will rebase this afterwards.

**Review by vaibhavsagar — APPROVED** [review]
LGTM! Happy to merge when compilation failures are resolved.

**Comment by teto** [comment]
Looks good

**Pull request merged**
